src/styles/normalizes.scss

```diff
@@ -0,0 +1,15 @@
+.catalog {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
```

Comment:
BEM block shouldn't have any margins. And as I mentioned before the body isn't a catalog in our case.

[STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/b6c30356-028d-49c4-8b35-9ceb6e990c6c)